Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -163,12 +163,10 @@ final class Analytics {
     }
 
     func showInvitePromo() {
-        func openInvitePromo() {
```

Comment:
nice find.. wondering how that snuck in 🙈

Reply:
My guess is somehow a merge conflict maybe.